communications/newsletters/newsletter_26_June2021.md

```diff
@@ -0,0 +1,175 @@
+
+This is an extended draft of an earlier chapter that describes what key documents should be added to a project repository.
+It now features examples from _The Turing Way_ repository and resources from the Open Life Science training program.
+-  Thanks to Lotty Coupet for writing [a case study from the AutSpaces project](https://the-turing-way.netlify.app/project-design/project-repo/project-repo-roadmapping.html) that describes how to set a roadmap for your project. 
+- Thanks to Andreea Avramescu, Helen Nicholls and Becki Green from the ‘Engage @ Turing ’ graduate programme for providing examples to [set up repositories for a software project](https://the-turing-way.netlify.app/project-design/project-repo/project-repo-advanced.html).
+
+![Maria quote tweeting The Turing Way announcement of new chapters: “That feeling when a great set of chapters (including a couple you've been working on) is about to get published in an amazing and open resource”](images/2021-06-mariatweet.png)
+
+*Tweet by [Maria Nantone](https://twitter.com/bynans1/status/1408568198002364429?s=20)*
+
+### [6 new chapters in the Guide for Communication](https://the-turing-way.netlify.app/communication)
+
+Emma Karoune, Maria Nanton and Marta Mangiarulo (our contributor in focus this week) began their collaboration at the Book Dash and ended up writing six chapters in the guide for communication that features personal stories from Veronika Cheplygina, Yanina Bellini, Emma Harris and Zoe Ingram.
+Read the following chapters to learn how to communicate your work with a wider audience
+  - [Overview of Guide for Communication](https://the-turing-way.netlify.app/communication/comms-overview.html)
+  - [Blogs for Research Communication](https://the-turing-way.netlify.app/communication/blogs.html)
+  - [Lay Summaries](https://the-turing-way.netlify.app/communication/lay-summaries.html)
+- [Podcasts for Research Communication](https://the-turing-way.netlify.app/communication/podcasts.html)
+- [Presenting Posters and Conference Talks](https://the-turing-way.netlify.app/communication/presentations.html)
+- [Social Media for Research Communications](https://the-turing-way.netlify.app/communication/social-media.html)
+
+## Contributor in Focus: Marta Mangiarulo
+
+![Picture of Marta. She is a white woman, with light brown hair. She is wearing a black sleeveless top and sunglasses. She is holding a red bottle with the soap solution and blowing soap bubbles]images/2021-06-marta.png)
+
+Marta Mangiarulo is a Research Assistant and Teaching Fellow in Psychology at the University of Leicester. 
+Her research interests involve probabilistic inference, social hypothesis testing, and reasoning biases. 
+She is also passionate about science communication and research dissemination and interested in replicability, open science issues, and the interface between cognitive and social aspects in social psychology. As the two times Book Dash attendee, Marta reflects on her participation:
```

Comment:
Should say - As a two time Book Dash attendee ...

Reply:
```suggestion
She is also passionate about science communication and research dissemination and interested in replicability, open science issues, and the interface between cognitive and social aspects in social psychology. As a two times Book Dash attendee, Marta reflects on her participation:
```